test/acceptance/behave/components/common_steps/batch_operations/update_query_steps.py

```diff
@@ -60,5 +61,22 @@ def update_entities_in_a_single_batch_operation(context, op):
         for row in context.table:
             queries_parameters[row["parameter"]] = row["value"]
 
-    context.resp = context.cb.batch_update(queries_parameters, op)
+    context.resp = context.cb.batch_update(queries_parameters, context.entities_accumulate, op)
     __logger__.info("...updated entities in a single batch operation")
+
+@step(u'update entities in a single batch operation "([^"]*)" in raw mode')
+def update_entities_in_a_single_batch_operation_in_raw_mode(context, op):
+    """
+    allows to create, update and/or delete several entities in a single batch operation in raw mode
+    used mainly with data type as boolean, dict, list, null, numeric, etc
+    :param context: It’s a clever place where you and behave can store information to share around. It runs at three levels, automatically managed by behave.
+    :param op: operations allowed (APPEND, APPEND_STRICT, UPDATE, DELETE)
```

Comment:
In the apiary reference in "Update Batch Operation" only show these actionTypes:
```
   actionType, to specify the kind of update action to do: either APPEND, APPEND_STRICT, UPDATE, DELETE.

```